parsers/LK.py

```diff
@@ -0,0 +1,107 @@
+#!/usr/bin/env python3
+
+from arrow import get
+from requests import Session
+
+from parsers import countrycode
+from parsers.lib import web
+
+
+def fetch_data(country_code, session=None):
+    countrycode.assert_country_code(country_code, 'LK')
+
+    html = web.get_response_soup(country_code, 'http://www.ceb.lk/yesterday-electricity/', session)
+    date = get(html.find_all('h4')[1].text.strip().split(':')[-1], 'MMMM D, YYYY')
+    head_vals = html.find_all('div', {'class': 'wpb_wrapper'})[3].find_all('div', {'class': 'row'})
+
+    demand = float(head_vals[0].text.strip().split('\n')[1].split(' ')[0])
+    storage = float(head_vals[1].text.strip().split('\n')[1].split(' ')[0])
+
+    rows = html.find_all('tr')[1:]
+
+    oil_value = wind_value = hydro_value = coal_value = 0.0
+
+    for row in rows:
+        values = row.text.lower().strip().split('\n')
+        if 'coal' in values[0]:
+            coal_value += float(values[1].split(' ')[0])
+        elif 'oil' in values[0]:
+            oil_value += float(values[1].split(' ')[0])
+        elif 'wind' in values[0]:
+            wind_value += float(values[1].split(' ')[0])
+        else:
+            hydro_value += float(values[1].split(' ')[0])
```

Comment:
Can you add an else if hydro and an else that throws an exception? This will be more robust

Reply:
I did that because every other power plant is hydroelectric power plant.

Reply:
Mmh but what would happen if they change that? You might not be around to maintain it so I prefer the parser being consistent as to what it expects. It will prevent future errors from being unnoticed.

Reply:
Sure, Ill make that change